content/2020-vancouver/index.md

```diff
@@ -16,6 +16,15 @@ Get an impression of PromCon EU 2019:
 
 Registration for PromCon NA 2020 will open soon.
 
+## Our sponsors
+
+A big thank you to all of our sponsors who are helping make PromCon NA 2020 happen!
```

Comment:
Should we add a link here to the page with the prospectus?

Reply:
No, that's linked from the sponsor page.